src/libraries/System.Resources.Extensions/tests/CompatTests/System.Resources.Extensions.Compat.Tests.csproj

```diff
@@ -0,0 +1,56 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-unix;</TargetFrameworks>
+    <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
+    <CustomResourceTypesSupport Condition="'$(EnableAggressiveTrimming)' == 'true' and '$(UseNativeAotRuntime)' != 'true'">true</CustomResourceTypesSupport>
+    <!-- some of the embeded resources have this assembly name hardcoded -->
+    <AssemblyName>System.Resources.Extensions.Tests</AssemblyName>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\**\*.cs" />
+    <Compile Remove="..\BinaryFormatTests\**\*.cs" />
```

Comment:
Don't glob.

Reply:
I am using glob on purpose here: it's going to include all new test files by default, ensuring that compatibility is checked for them as well. And it also makes my life easier, as I don't have to manually list all the files and make sure VS does not change the whole project file anytime I rename one of them.

Reply:
In my personal opinion, being explicit in a sources file is part of good engineering.  It does look like there are two other test projects using globs, so I can't complain as to you being the first one to do it.